compiler/cli/src/main/core/com/kitakkun/backintime/compiler/cli/BackInTimeCommandLineProcessor.kt

```diff
@@ -12,21 +12,30 @@ import org.jetbrains.kotlin.config.CompilerConfigurationKey
 class BackInTimeCommandLineProcessor : CommandLineProcessor {
     companion object {
         private const val KEY_ENABLED = "enabled"
+        private const val KEY_CONFIG_FILE = "config"
         val CONFIG_KEY_ENABLED = CompilerConfigurationKey.create<Boolean>(KEY_ENABLED)
+        val CONFIG_KEY_CONFIG_FILE = CompilerConfigurationKey.create<String>(KEY_CONFIG_FILE)
     }
 
-    override val pluginId: String = ""
+    override val pluginId: String = "com.kitakkun.backintime.compiler"
     override val pluginOptions: Collection<AbstractCliOption> = listOf(
         CliOption(
             optionName = KEY_ENABLED,
             valueDescription = "true|false",
             description = "Whether BackInTime plugin is enabled or not.",
             required = false,
         ),
+        CliOption(
+            optionName = KEY_CONFIG_FILE,
+            valueDescription = "path/to/backintime-config.yaml",
+            description = "configuration file for backintime compiler",
+            required = false,
+        ),
```

Comment:
This CliOption should be configured via BackInTimeExtension, that is responsible for configuring compiler in the Gradle DSL.

Reply:
[Make config file path configurable via gradle (not tested)](https://github.com/kitakkun/back-in-time-plugin/pull/138/commits/7d9074677c5797c5a0d192c1deeb447b139ae4b3)

not tested, but implemented.